src/version.ts

```diff
@@ -1,6 +1,5 @@
-'use strict'
```

Comment:
Is there a reason this was removed?

Reply:
Deno enforces strict mode automatically https://github.com/denoland/deno/issues/5845